profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/features/DataCenterApiMockTest.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.jclouds.profitbricks.rest.features;
+
+import com.squareup.okhttp.mockwebserver.MockResponse;
+import java.util.List;
+import org.apache.jclouds.profitbricks.rest.domain.DataCenter;
+import org.apache.jclouds.profitbricks.rest.domain.Location;
+import org.apache.jclouds.profitbricks.rest.domain.options.DepthOptions;
+import org.apache.jclouds.profitbricks.rest.internal.BaseProfitBricksApiMockTest;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+import org.testng.annotations.Test;
+
+@Test(groups = "unit", testName = "DataCenterApiMockTest", singleThreaded = true)
+public class DataCenterApiMockTest extends BaseProfitBricksApiMockTest {
+   
+   @Test
+   public void testGetList() {
+      server.enqueue(
+         new MockResponse().setBody(stringFromResource("/datacenter/list.json"))
+      );
+      
+      List<DataCenter> list = dataCenterApi().list(new DepthOptions().depth(1));
+      
+      assertNotNull(list);
+      assertEquals(list.size(), 3);
+      assertEquals(list.get(0).properties().name(), "vea");
```

Comment:
You need to verify the recorded request. MockWebServer will return each enqueued request upon *any* request. You need to verify that the generated request is the expected one: expected request uri, method, headers and body.

The same must be applied in all methods in this class.